00-Introducing-HOOMD-blue/06-Equilibrating-the-System.ipynb

```diff
@@ -0,0 +1,747 @@
+{
```

Comment:
<p>class name.</p><p> </p><p>Also, this is fine to do for the interim while we decide how to handle operation completion. On the other hand, we could use a loop and check after every so many steps for completion and then manual remove it. I am not sure what is the better example to set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I think that tuning during a fixed period at beginning and the continuing with that move size is the most effective way to run simulations. I am not convinced that stopping the move size tuner when it achieves an acceptance ratio with a tolerance. The acceptance ratio is going to fluctuate due to the random trial moves.</p>